packages/react-component-library/src/index.ts

```diff
@@ -30,4 +35,9 @@ export {
   Tab,
   TabSet,
   TextInput,
+
+  Icons,
```

Comment:
Why have spaces and can we just do it alphabetically to make it easier to find things and avoid possible duplication?

Reply:
The exports are organised in a way that corresponds with the imports at the top of the file.

Happy to make it all alphabetical though.

Reply:
I think when we move to the alternative export method I mention we do it then